.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     // some of these rules going forwards to clean up the code base.
     'import/no-extraneous-dependencies': 0,
     'import/no-unresolved': 0,
+    'import/extensions': 0,
```

Comment:
Using ES6 import syntax, I was getting lint errors because I think you need to configure this import plugin to understand what's going on when you alias modules, like what is done with 'kolibri'.